src/main/java/pw/itr0/kaba/encrypt/PasswordUtil.java

```diff
@@ -4,9 +4,42 @@
 import java.util.Arrays;
 import java.util.Objects;
 
-public class PasswordUtil {
+/**
+ * Encryption utility for password of vaults, such as the password used to unlock the keystore.
+ * <p>
+ * Encrypt / decrypt password algorithm using PBE.
+ *
+ * @author ryotan
+ * @since 1.0.0
+ */
+public final class PasswordUtil {
+
+    /**
+     * Error message for null password argument.
+     */
+    private static final String PASSWORD_MUST_NOT_BE_NULL = "password must not be null";
+
+    /**
+     * Prefix of sealed password string.
+     */
+    private static final String SEALED_PASSWORD_PREFIX = "SEALED:V1:";
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The String "SEALED:V1:" appears 2 times in the file.](https://www.codacy.com/app/ryotan/kaba-vault/file/2578707355/issues/source?bid=3298589&fileBranchId=3301306#l25)